src/Context.php

```diff
@@ -105,4 +105,37 @@ private static function get_campaign_scope( $global_project ) {
 				return 'Global';
 		}
 	}
+
+	/**
+	 * @param array       $context   Context to be set.
+	 * @param array       $meta      Meta data.
+	 * @param string|null $post_type Post type.
+	 */
+	public static function set_custom_styles(
+		array &$context,
+		array $meta,
+		?string $post_type = null
+	): void {
+		if ( 'campaign' === $post_type ) {
+			$custom_styles = [
+				'nav_type'            => $meta['campaign_nav_type'] ?? null,
+				'nav_border'          => $meta['campaign_nav_border'] ?? null,
+				'campaign_logo_color' => 'green',
```

Comment:
Just a doubt, but we're going to use the default green color? Or we should use the Greenpeace's green color?

Reply:
This part is the legacy code for the Campaigns (a copy/paste of [this part](https://github.com/greenpeace/planet4-master-theme/pull/1633/files/c0f9f58d17d737a44cb8bb6a28e28f43e33d2854#diff-9aee92185c5b9b4bd23f4060ede93e8ea8b08c18b8b56643884c5ac8f31b118dL73-L86)), I think it should stay like that; for the Actions I assume the colors are controlled by the theme itself